### PR TITLE
Make PR template clearer

### PR DESCRIPTION
**Pull request by m1guelpf**

### CODE REVIEW

**Comment by caxvis** [comment]
I think a dialog text would be clearer instead of using the `{question}? {answer} template.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
   Q. Which branch should I use for my pull request?
-  A. Use `master` branch (probably).
-
-  Q. Which branch if my change is a bug fix for Gatsby v1?
-  A. In this case, you should use the `v1` branch
-
-  Q. Which branch if I'm still not sure?
-  A. Use `master` branch. Ask in the PR if you're not sure and a Gatsby maintainer will be happy to help :)
+  - Adding a new feature? Use `master` branch
+  - Fixing a bug for Gastby v1? Use the `v1` branch
```
Comment (amberleyromo) [inline .github/PULL_REQUEST_TEMPLATE.md]: Also "Gastby" to "Gatsby" :)

**Comment by pieh** [comment]
> I'd almost rather we remove this section as v2 continues to grow in adoption and becomes the standard.

Yeah, I do agree.

**Comment by LekoArts** [comment]
@pieh This could be merged, right?

**Comment by jgierer12** [comment]
Since `master` is the default branch anyways, wouldn't it be better to completely remove the section about the branch?

**Comment by pieh** [comment]
> Since `master` is the default branch anyways, wouldn't it be better to completely remove the section about the branch?

There are multiple ways repositories (and releases) can be organized. Some will use `master` for releases and branches like `dev` is where PRs should be target at. I don't see much harm in keeping information to open PRs against `master` branch - with this PR it is concise and quickly readable for first time contributors.

**Pull request merged**